pylint_ignore.py

```diff
@@ -0,0 +1,17 @@
+""" Gratefully copied from https://github.com/PyCQA/pylint/issues/2686#issuecomment-621927895. """
```

Comment:
please dont put this in the root dir, rather in Pyrado/utils or similar

Reply:
See #39.